cg/apps/tb/api.py

```diff
@@ -49,6 +52,32 @@ def parse_qcmetrics(data: dict) -> dict:
         """Call internal Trailblazer MIP API."""
         return files.parse_qcmetrics(data)
 
+    def is_analysis_ongoing(self, case_id: str) -> bool:
+        """Call internal Trailblazer API"""
+        return self.is_ongoing(family=case_id)
+
+    def is_analysis_failed(self, case_id: str) -> bool:
+        """Call internal Trailblazer API"""
+        return self.is_failed(family=case_id)
+
+    def is_analysis_completed(self, case_id: str) -> bool:
+        """Call internal Trailblazer API"""
+        return self.is_completed(family=case_id)
+
+    def has_analysis_started(self, case_id: str) -> bool:
+        """Check if analysis has started"""
+        if self.is_analysis_ongoing(case_id=case_id):
+            LOG.warning("%s: analysis is ongoing - skipping", case_id)
```

Comment:
These logs should probably not say "skipping", the API can't know that the calling function will "skip" them. 

It's a bit weird that the API logs warnings on these things, why would it? I think I know why, you want to preserve the previous logic :) I think we should not log these things here, better log in the calling function, we'll lose some details but it will be a nicer API.

Reply:
Good point! Done